### PR TITLE
Fix nesting in pairarrow context

### PR DESCRIPTION
**Pull request by t-bltg**
Fix https://github.com/domluna/JuliaFormatter.jl/issues/453.

### CODE REVIEW

File: test/default_style.jl [hunk 1]
```diff
@@ -2878,6 +2878,15 @@
...
         @test fmt(str, 4, 1) == str
+
+        # https://github.com/domluna/JuliaFormatter.jl/issues/453
+        str = """
+        bar = Dict(
+            :foo => \"""A triple quoted literal string
+                    with some words in it.\""",
+        )
+        """
+        @test fmt(fmt(str, 4, 20), 4, 20) == str
```
Comment (domluna) [inline test/default_style.jl:2889]: how come this calls `fmt` twice?

Comment (domluna) [reply test/default_style.jl:2889]: fmt already formats things twice btw, fmt1 is equivalent to calling format_text once

Comment (t-bltg) [reply test/default_style.jl]: ok, fixed

**Comment by t-bltg** [comment]
Thanks :+1:

**Pull request merged**